docs/visual-testing/integrations/csharp.md

```diff
@@ -0,0 +1,319 @@
+---
+sidebar_label: C#/.Net
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# C#/.Net WebDriver Integration
+
+:::note Important
+Access to this feature is currently limited to Enterprise customers as part of our commitment to providing tailored solutions. We are excited to announce that self-service access is under development and will be released shortly. Stay tuned!
+:::
+
+## Introduction
+
+This guide requires an existing C# NUnit / xUnit setup.<br />
+You can alternatively take a look to our [example repository](#examples).
+
+Sauce Visual provides an library allowing integration with [WebDriver](https://www.selenium.dev/documentation/webdriver/).
+
+Sauce Visual plugin provides a library exposing a `VisualClient` object that provides actions:
+
+- `visualClient.VisualCheck()`: Takes a screenshot and send it to Sauce Visual for comparison.
+- `visualClient.VisualResults()`: Waits for all diff calculations to complete and returns a summary of results.
+  See [Test results summary](#test-results-summary) for more details about summary format and sample usage.
+
+## Quickstart
+
+### Step 1: Add Sauce Visual dependecy
+
+Add [Sauce Visual](https://www.nuget.org/packages/SauceLabs.Visual/) dependency to your `.csproj`
```

Comment:
normally you don't put such a code to show how to integrate - I recommend using nuget/dotnet reference instead:

```
Windows NuGet CLI:

`Install-Package SauceLabs.Visual`

Linux/Mac OS X .NET Core CLI:

`dotnet add package SauceLabs.Visual`
```